base/dcp_client_metadata.go

```diff
@@ -255,9 +255,9 @@ func (m *DCPMetadataCS) Persist(workerID int, vbIDs []uint16) {
 
 func (m *DCPMetadataCS) load(workerID int) {
 	var meta WorkerMetadata
-	_, err := m.bucket.Get(m.getMetadataKey(workerID), &meta)
+	_, err := m.collection.Get(m.getMetadataKey(workerID), &meta)
 	if err != nil {
-		if IsKeyNotFoundError(m.bucket, err) {
+		if IsDocNotFoundError(err) {
```

Comment:
I don't think the behaviour of IsKeyNotFoundError and IsDocNotFoundError are equivalent.  Is this an intentional switch to use IsDocNotFoundError?

Reply:
This is another question for @bbrks I suppose - `IsKeyNotFoundError` is not implemented for collections.

Reply:
When using a Bucket, I keep this behavior.